packages/core/src/routes/+page.svelte

```diff
@@ -566,6 +566,13 @@ const onHoverDelayMsInput = (event: Event) => {
     </Button>
 
     <Button progress="indeterminate">Loading</Button>
+    <IconButton
+      label="Pause"
+      variant="secondary"
+      progress="indeterminate"
+      icon="pause"
```

Comment:
Is there a good way to type this so we don't need (or are prevented from) passing an icon if we also pass progress?

This feels surprising -- but not important

Reply:
I get the surprising part, but I don't think that we'd want to make these props mutually exclusive. The most efficient way to use this component is like this because no expensive unmounting / remounting of HTML elements occurs:

```
<script>
  $: progress = loading ? 'indeterminate' : undefined
</script>

<IconButton
  {progress}
  icon='pause'
/>

Reply:
Ahh, I hadn't considered that. Neat